### ISSUE TITLE
DeprecationWarning for Python 3.12

### ISSUE THREAD

**Issue by gdalmau** [rsinger86/django-lifecycle#90]
Python 3.10 shows the following warning regarding a future deprecation in Python 3.12:

```shell
  /usr/local/lib/python3.10/site-packages/django_lifecycle/django_info.py:1: DeprecationWarning: The distutils package is deprecated and slated for removal in Python 3.12. Use setuptools or check PEP 632 for potential alternatives
    from distutils.version import StrictVersion
```

It would be nice to fix it at some point.

**Comment by nextmat** [comment]
I'm seeing this as well. To clarify, the deprecation warning is with Django 3.2.12.

I'd be happy to put a PR together if a maintainer can point me in the right direction.

**Comment by nextmat** [comment]
I'm running into this as well, opened a PR with fix in #93.

**Comment by gdalmau** [comment]
Thanks @nextmat! Closing this issue since it's fixed by #93
